nvsvocprez/app.py

```diff
@@ -1901,16 +1922,25 @@ def _render_puv_rdf(self):
             }
             WHERE {
               <xxx> ?p ?o .
-
-              # include only PUV properties
-              # FILTER (STRSTARTS(STR(?p), "https://w3id.org/env/puv#"))
+              FILTER ( ?p != skos:broaderTransitive )
+              FILTER ( ?p != skos:narrowerTransitive )
+              FILTER ( ?p != skos:broader )
+              FILTER ( ?p != skos:narrower )
+              FILTER ( ?p != skos:related )
+              FILTER ( ?p != owl:sameAs )
+              # exclude other properties from altprof
+              <EXCLUDE_FILTERS>
             }
             """.replace(
             "xxx", self.instance_uri
+        ).replace(
+            "<EXCLUDE_FILTERS>", "\n".join(exclude_filters)
```

Comment:
You can convert this to an f string and using double braces to escape the any single braces within the query. 

i.e.

```python
"""
WHERE {{ 
              FILTER ( ?p != skos:broaderTransitive )
              FILTER ( ?p != skos:narrowerTransitive )
              FILTER ( ?p != skos:broader )
              FILTER ( ?p != skos:narrower )
              FILTER ( ?p != skos:related )
              FILTER ( ?p != owl:sameAs )
              # exclude other properties from altprof
              { exclude_filters }
}}
"""
```